utils/swift_build_support/swift_build_support/products/playgroundsupport.py

```diff
@@ -0,0 +1,131 @@
+# swift_build_support/products/playgroundsupport.py -------------*- python -*-
+#
+# This source file is part of the Swift.org open source project
+#
+# Copyright (c) 2014 - 2017 Apple Inc. and the Swift project authors
+# Licensed under Apache License v2.0 with Runtime Library Exception
+#
+# See https://swift.org/LICENSE.txt for license information
+# See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+#
+# ----------------------------------------------------------------------------
+
+import os
+import re
+from . import product
+from .. import shell
```

Comment:
If possible I would like for us to use the new `build_swift.shell` module. It should have all the functionality you want and is Python 2/3 compatible.

Reply:
Actually we can hold-off for now and do follow-up PRs to migrate to the new `shell` module.